ranch/roadway.py

```diff
@@ -1152,7 +1165,8 @@ def build_taz_drive_connector(
         # geometries (not reference) - good intersections
 
         taz_good_intersection_df = Roadway.get_nodes_in_zones(
-            node_two_geometry_df, taz_polygon_df
+            node_two_geometry_df.drop(columns=["index_left", "index_right"], errors="ignore"), 
```

Comment:
Do we need this? If these two columns exist in the MTC workflow, I'd rather clean up the MTC data before calling this method. We should keep Ranch as generic as possible.